velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -451,7 +452,10 @@ void WindowFuzzer::go() {
     auto useRowNumberKey =
         requireSortedInput || windowType == core::WindowNode::WindowType::kRows;
 
-    const auto partitionKeys = generateSortingKeys("p", argNames, argTypes);
+    const uint32_t numKeys =
+        boost::random::uniform_int_distribution<uint32_t>(1, 15)(rng_);
+    const auto partitionKeys =
+        generateSortingKeys("p", argNames, argTypes, false, numKeys);
```

Comment:
nit: It looks like the only difference from before is the max cap of numKeys (5 vs. 15). Could we pass this max cap instead of numKeys to generateSortingKeys, then WindowFuzzer and AggregationFuzzer can pass 15 and 5 individually?

Reply:
@kagamiori : Sorry for missing out on this. WIll send out a PR.